### ISSUE TITLE
Improve formatting of instances added manually

### ISSUE THREAD

**Issue by LiLoDavis** [microsoft/accessibility-insights-web#822]
## Problem
For Assessment > Parsing, when I copy a failure from the Nu HTML checker into Accessibility Insights for web, the pasted content is poorly formatted.

For example, a failure that looks like this in the Nu HTML checker:
![image](https://user-images.githubusercontent.com/45569509/59702647-058ce400-91ad-11e9-814f-858897278317.png)

...looks like this when first pasted into Accessibility Insights for Web:
![image](https://user-images.githubusercontent.com/45569509/59702703-281efd00-91ad-11e9-8505-1e8d8b2703a8.png)

...and like this in a report:
![image](https://user-images.githubusercontent.com/45569509/59702826-7af8b480-91ad-11e9-9f9c-90637b616f0e.png)

## Desired outcome
The failure instance should retain at least the line breaks (as in second screenshot). Even better to provide appropriate styling:

#### Error: Start tag seen without seeing a doctype first. Expected `<!DOCTYPE html>`. 
From line 1, column 1; to line 1, column 6 
`<html><head>`

**Comment by adarshrema** [comment]
Hi @LiLoDavis, thank you for your interest in Accessibility Insights and this feature request. This was triaged by the team and has been added to our backlog. In addition, this has been marked as good first issue and help wanted to encourage community contribution.

**Comment by fxnoob** [comment]
Can I work on this issue?

**Comment by ferBonnin** [comment]
Hi @fxnoob for sure! 
Please take a look at our[ README file](https://github.com/microsoft/accessibility-insights-web/blob/master/README.md) for instructions on how to build and test

**Comment by tohmjudson-ms** [comment]
@ferBonnin This seems to still be active, mind if I work on it as a first bug?

**Comment by ferBonnin** [comment]
Hi @tohmjudson-ms for sure! I am assigning this to you

thanks!
